tensorflow/lite/delegates/webnn/webnn_delegate.cc

```diff
@@ -240,9 +245,18 @@ class Subgraph {
           std::string name = std::to_string(t);
           operand = wnn_builder.call<emscripten::val>("input", name, desc);
         } else {
-          auto data_size = context->tensors[t].bytes / 4;
-          emscripten::val view{ emscripten::typed_memory_view(data_size, static_cast<const float*>(data)) };
-          operand = wnn_builder.call<emscripten::val>("constant", desc, view);
+          if (dims.size() == 0) {
```

Comment:
Do we still need this branch? At line 238, you set `dims = {1}` when `dims.size() == 0`.

Reply:
I planned to remove this workaround once we have single-value constant supported in Chromium.

Reply:
Sounds good, please add a TODO for that. Thanks!